src/pudl/metadata/fields.py

```diff
@@ -196,6 +202,25 @@
     "asset_type": {
         "type": "string",
         "description": "Type of asset being reported to the core_ferc1__yearly_balance_sheet_assets_sched110 table.",
+        "constraints": {
+            "enum": ASSET_TYPES_FERC1.extend(
+                # Add all possible correction records into enum
```

Comment:
not blocking at all: for all of these corrections, it am tempted to suggest adding a correction for all of them. i know not all factoids get corrections because not all factoids are calculated (and even within those not all of them get corrections bc some calculated fields are surprisingly clean). but i see why you wouldn't do this and don't feel strongly about it either way.

```python
ASSET_TYPES_FERC1 + ["{t}_correction" for t in ASSET_TYPES_FERC1]
```